tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2020",
```

Comment:
Lets wait until X PR gets merged, but I'm afraid you are going to have to change this to es2019, and set it to es2020 in the rollup config, like @diegopf did in the other PR.

Reply:
This is going to be far more complicated here, probably we will need to configure babel for webpack, as X-Components are exporting modern code